### PR TITLE
helm: release notes for 5.5.0

### PR DESCRIPTION
**Pull request by narqo**
#### What this PR does

I wrote the notes based on 5.5.0-rc.0 changelog #9487. There aren't much of changes in the release and I tried to focus on those, that potentially affect the users.

#### Which issue(s) this PR fixes or relates to

Relates to #9382 

#### Checklist

- [ ] Tests updated.
- [ ] Documentation added.
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`.
- [ ] [`about-versioning.md`](https://github.com/grafana/mimir/blob/main/docs/sources/mimir/configure/about-versioning.md) updated with experimental features.

### CODE REVIEW

File: docs/sources/helm-charts/mimir-distributed/release-notes/v5.5.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+## Important changes
+
+The ingester is configured with `GOMAXPROCS` to help with the overhead from Go scheduler, observed on hosts with lots of CPU cores.
+
+The GEM query-frontend logs the query author's access policy name and token name in the query statistics logs. This provides a better visibility of who (which token) is running a potentially expensive query.
+
+## Features and enhancements
+
+Notable enhancements are as follows:
```
Comment (lamida) [inline docs/sources/helm-charts/mimir-distributed/release-notes/v5.5.md:22]: Just formatting suggestion, should we turn the lines below into bulleted list? The same comment applies to the "Improtant changes" section above.

Comment (narqo) [reply docs/sources/helm-charts/mimir-distributed/release-notes/v5.5.md:22]: Sure, I reformatted the text. PTAL

File: docs/sources/helm-charts/mimir-distributed/release-notes/v5.5.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+# Grafana Mimir Helm chart version 5.5 release notes
+
+Grafana Labs is excited to announce version 5.5 of the Grafana Mimir Helm chart, which is compatible with Grafana Mimir v2.14 or Grafana Enterprise Metrics (GEM) v2.14. The `mimir-distributed` Helm chart is the best way to install Mimir on Kubernetes.
+
+The highlights that follow include the top features, enhancements, and bug fixes in this release. For a comprehensive list of changes, see the [Helm chart changelog](https://github.com/grafana/mimir/tree/main/operations/helm/charts/mimir-distributed/CHANGELOG.md).
+
+## Important changes
+
+- The ingester is configured with `GOMAXPROCS` to help with the overhead from Go scheduler, observed on hosts with lots of CPU cores.
+- The GEM query-frontend logs the query author's access policy name and token name in the query statistics logs. This provides a better visibility of who (which token) is running a potentially expensive query.
```
Comment (dimitarvdimitrov) [inline docs/sources/helm-charts/mimir-distributed/release-notes/v5.5.md:17]: doesn't this belong to the GEM changelog? or was it a change to the chart?

**Review by dimitarvdimitrov — COMMENTED** [review]
LGTM, I'd wait for an approval from @tacole02 too

**Review by dimitarvdimitrov — APPROVED** [review]

File: docs/sources/helm-charts/mimir-distributed/release-notes/v5.5.md (hunk 2 above)
Comment (narqo) [reply docs/sources/helm-charts/mimir-distributed/release-notes/v5.5.md:17]: We have this in GEM changelog also. This one is about making the feature configured and enabled by default in the chart.

File: docs/sources/helm-charts/mimir-distributed/release-notes/v5.5.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+# Grafana Mimir Helm chart version 5.5 release notes
+
+Grafana Labs is excited to announce version 5.5 of the Grafana Mimir Helm chart, which is compatible with Grafana Mimir v2.14 or Grafana Enterprise Metrics (GEM) v2.14. The `mimir-distributed` Helm chart is the best way to install Mimir on Kubernetes.
+
+The highlights that follow include the top features, enhancements, and bug fixes in this release. For a comprehensive list of changes, see the [Helm chart changelog](https://github.com/grafana/mimir/tree/main/operations/helm/charts/mimir-distributed/CHANGELOG.md).
+
+## Important changes
+
+- The ingester is configured with `GOMAXPROCS` to help with the overhead from Go scheduler, observed on hosts with lots of CPU cores.
```
Comment (tacole02) [inline docs/sources/helm-charts/mimir-distributed/release-notes/v5.5.md:16]:
```suggestion
- The ingester is configured with `GOMAXPROCS` to help with overhead from the Go scheduler, observed on hosts with a large number of of CPU cores.
```

File: docs/sources/helm-charts/mimir-distributed/release-notes/v5.5.md (hunk 2 above)
Comment (tacole02) [inline docs/sources/helm-charts/mimir-distributed/release-notes/v5.5.md:17]:
```suggestion
- The GEM query-frontend logs the query author's access policy name and token name in the query statistics logs. This provides more visibility as which token is running a potentially expensive query.
```

File: docs/sources/helm-charts/mimir-distributed/release-notes/v5.5.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+- The GEM query-frontend logs the query author's access policy name and token name in the query statistics logs. This provides a better visibility of who (which token) is running a potentially expensive query.
+- Dashboards are updated to allow switching between classic and native histograms in the queries that support it.
+
+## Features and enhancements
+
+Notable enhancements are as follows:
+
+- The dashboard config maps now support setting a namespace.
```
Comment (tacole02) [inline docs/sources/helm-charts/mimir-distributed/release-notes/v5.5.md:24]:
```suggestion
- The dashboard configuration maps support setting a namespace.
```

File: docs/sources/helm-charts/mimir-distributed/release-notes/v5.5.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+- The `topologySpreadConstraints` configuration of all components is updated to better match what is allowed in Kubernetes.
+- The metamonitoring now supports AWS Signature Version 4 (sigv4) authentication for remote write.
```
Comment (tacole02) [inline docs/sources/helm-charts/mimir-distributed/release-notes/v5.5.md:26]:
```suggestion
- Metamonitoring supports AWS Signature Version 4 (sigv4) authentication for remote write.
```

File: docs/sources/helm-charts/mimir-distributed/release-notes/v5.5.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+- The metamonitoring is updated allowing its Grafana Agent pods setting image pull secrets, node selectors, tolerations, topology spread constraints, and resource requests and limits.
```
Comment (tacole02) [inline docs/sources/helm-charts/mimir-distributed/release-notes/v5.5.md:27]:
```suggestion
- Metamonitoring is updated to allow its Grafana Agent pods setting image pull secrets, node selectors, tolerations, topology spread constraints, and resource requests and limits.
```

Comment (tacole02) [reply docs/sources/helm-charts/mimir-distributed/release-notes/v5.5.md:27]: `setting` = `to set`? I'm not sure what the intended meaning of this point is, but it reads somewhat unclear.

**Review by tacole02 — APPROVED** [review]
Looks great! I left a few minor suggestions and one question for clarification.

File: docs/sources/helm-charts/mimir-distributed/release-notes/v5.5.md (hunk 6 above)
Comment (narqo) [reply docs/sources/helm-charts/mimir-distributed/release-notes/v5.5.md:27]: Yes

```suggestion
- Metamonitoring is updated to allow its Grafana Agent pods to set image pull secrets, node selectors, tolerances, topology spread constraints, and resource requests and limits.
```

**Pull request merged**